jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/workitem/RepositoryStorageVFSImpl.java

```diff
@@ -72,35 +70,30 @@ public RepositoryStorageVFSImpl( @Named("configIO") final IOService ioService, @
     @SuppressWarnings("unchecked")
     @PostConstruct
     public void init() {
-        configPath = fileSystem.getPath( "service-tasks", "remote", "service-repository-config.xml" );
+        configPath = fileSystem.getPath("service-tasks",
```

Comment:
@adrielparedes could you please keep the formating here?

Reply:
+1 there are quite a few formatting changes that are not needed.